lib/hl7/query.ex

```diff
@@ -48,12 +48,19 @@ defmodule HL7.Query do
 
   defstruct selections: [], invalid_message: nil, part: nil
 
+  @deprecated "use HL7.Query.sigil_f/2 instead"
```

Comment:
```suggestion
  @deprecated "use HL7.Query.sigil_p/2 instead"
```

Reply:
Good catch. I can update that and clean up the branch history at the same time.